packages/authorize/src/api.ts

```diff
@@ -0,0 +1,102 @@
+import { avUserPermissionsApi, avRegionsApi } from '@availity/api-axios';
+
+import type { Permission, RequestedPermissions, RequestedResources } from './types';
+
+export const getRegion = async (region?: boolean | string): Promise<string | undefined> => {
```

Comment:
I think we can take advantage of [useCurrentRegion](https://github.com/Availity/availity-react/blob/master/packages/hooks/src/useCurrentRegion.js) here